curp/src/server/raw_curp/mod.rs

```diff
@@ -284,6 +289,9 @@ impl<C: 'static + Command, RC: RoleChange + 'static> RawCurp<C, RC> {
         if st_r.role != Role::Leader {
             return Err(CurpError::Redirect(st_r.leader_id, st_r.term));
         }
+        if self.lst.get_transferee().is_some() {
+            return Err(CurpError::LeaderTransfer("leader transferring".to_owned()));
+        }
```

Comment:
Some other methods such as `handle_propose_conf_change` and `handle_publish` seems also needs to return an error on leader transfering.